jobs/release.Jenkinsfile

```diff
@@ -285,11 +285,12 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
                 // Run plume to publish official builds; This will handle modifying
                 // object ACLs, modifying AMI image attributes,
                 // and creating/modifying the releases.json metadata index
+                def (bucket, subpath) = pipeutils.splitBucketKey(s3_stream_dir)
                 shwrap("""
                 cosa shell -- plume release --distro fcos \
                     --version ${params.VERSION} \
                     --stream ${params.STREAM} \
-                    --bucket ${pipecfg.s3_bucket} \
+                    --bucket ${bucket} \
```

Comment:
underneath the covers what is plume doing with this bucket?

It feels a bit like we're reverse engineering the change we're making here and it feels a bit like the relationship here isn't clean.

Reply:
It ends up doing only things actually under the `prod/streams/${STREAM}` key. It could make sense to have `--bucket` take the key too, but I didn't want to tackle that right now. I might revisit this when looking at the whole `plume` call soon since that's another delta we need to parameterize.

Reply:
Maybe call out the elephant in the room with a comment then?